job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/ScheduledExecutorConfig.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.scheduled.executor;
+
+/**
+ * Configuration class for the Job Service Scheduled Executor. Includes connection properties to both database and RabbitMQ.
+ */
+public class ScheduledExecutorConfig {
+
+    public static String getDatabaseURL(){
```

Comment:
This module is crying out for a `getPropertyOrEnvVar()` utility function.

Probably the same as this one:
https://github.houston.softwaregrp.net/caf/caf-storage/commit/058bdef00447588551e4e2c8f1560211ae4b15a5